sdcm/utils/k8s/chaos_mesh.py

```diff
@@ -44,3 +67,106 @@ def initialize(self) -> None:
             timeout="30m"
         )
         LOGGER.info("chaos-mesh installed successfully on %s k8s cluster.", self._k8s_cluster.k8s_scylla_cluster_name)
+
+
+class ExperimentStatus(Enum):
+    STARTING = 0
+    RUNNING = 1
+    PAUSED = 2
+    FINISHED = 3
+    ERROR = 4
+    UNKNOWN = 5
+
+
+class PodChaosExperiment:
+    """Base class for all PodChaos experiments."""
+    API_VERSION = "chaos-mesh.org/v1alpha1"
+
+    def __init__(self, pod: "BasePodContainer", name: str):
```

Comment:
Requiring a `pod` here looks wrong. You use `db_cluster` object only out of it. So, why not using `db_cluster` param directly?

Reply:
I seem like it's used for targeting, I would expect the targeting code to be in one place.

When we target a specific pod, or when we target the whole cluster.
I think the most common case, is targeting a single pod, and we should built is so we don't need to repeat those again and again

Reply:
true, fixed